### PR TITLE
add link to history ("archive") in generated pages

### PR DESCRIPTION
**Pull request by dobbymoodge**

### CODE REVIEW

**Comment by dobbymoodge** [comment]
@danmcp

File: templates/developers_overview.erb [hunk 1]
```diff
@@ -246,7 +246,7 @@ Next: <%=next_sizing_total%><br />
     <br/>
     <br/>
     <hr>
-    <i>Generated at <%=Time.now.strftime("%I:%M%p %Z")%></i>
+    <i>Generated at <%=Time.now.strftime("%I:%M%p %Z")%></i> - <a href="/history/">archives</a>
```
Comment (danmcp) [inline templates/developers_overview.erb]: How about archive_path

**Comment by dobbymoodge** [comment]
Depends on https://github.com/openshift/trello_config/pull/11

File: templates/developers_overview.erb (hunk 1 above)
Comment (dobbymoodge) [inline templates/developers_overview.erb]: @danmcp good. fixed.

**Comment by dobbymoodge** [comment]
@danmcp I've tested this and the linked dependency and it works well locally.

**Pull request merged**